lte/gateway/c/core/oai/test/n11/test_auth_service_client.cpp

```diff
@@ -99,6 +102,42 @@ TEST(test_get_subs_auth_info, get_subs_auth_info) {
   EXPECT_TRUE(snni == req.serving_network_name());
 }
 
+TEST(
+    test_convert_proto_msg_to_itti_m5g_auth_info_ans,
+    convert_proto_msg_to_itti_amf_decrypted_imsi_info_ans) {
+  magma::lte::M5GSUCIRegistrationAnswer response;
+  itti_amf_decrypted_imsi_info_ans_t amf_decrypt_imsi_info_resp;
+
+  std::string imsi("1032547698");
+
+  response.set_ue_msin_recv(imsi);
+
+  magma5g::convert_proto_msg_to_itti_amf_decrypted_imsi_info_ans(
+      response, &amf_decrypt_imsi_info_resp);
+
+  EXPECT_TRUE(
```

Comment:
take care of github actions comment. you can convert to std::string for each string item and use EXPECT_EQ.

Reply:
Done